helm/aladdin-demo/templates/aladdin-demo.deploy.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    name: {{ .Chart.Name }}
+spec:
+  replicas: {{.Values.deploy.replicas}}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        deploymentconfig: {{ .Chart.Name }}
+    spec:
+      terminationGracePeriodSeconds: {{ .Values.deploy.terminationGracePeriod }}
```

Comment:
Maybe add a comment explaining what this is. Most projects probably won't need this.